libs/mrwebrtc/include/interop_api.h

```diff
@@ -195,22 +258,45 @@ using mrsVideoCaptureFormatEnumCallback = void(
 using mrsVideoCaptureFormatEnumCompletedCallback =
     void(MRS_CALL*)(void* user_data, mrsResult result);
 
-/// Enumerate the video capture formats asynchronously.
+/// Asynchronously enumerate the capture formats of a video capture device.
+///
+/// The capture formats are enumerated for the given video capture device:
+/// - If the device does not support video profiles, |profile_id| and
+/// |profile_kind| are both ignored, and all capture formats supported by the
+/// specified device are enumerated.
+/// - If the device supports video profiles, enumeration depends on the values
+/// of the profile ID and kind. The profile ID string |profile_id| can be null
+/// or an empty string, meaning no specified profile ID. The profile kind
```

Comment:
`null` as "unspecified" is standard, but empty string is a bit weird. I'd rather just check for null here and treat empty string as any other string.

Reply:
That makes sense indeed.

Reply:
Actually this is annoying because we convert to `absl::string_view` so lose the distinction. Also in general I don't think we make much difference between `nullptr` and empty strings, I think we use the `IsStringNullOrEmpty()` helper on most string arguments in the API, and since there's no empty string profile ID then I would be for treating it the same as `nullptr`.